source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -45,6 +59,10 @@ bool BioXASSideAppController::setupDataFolder()
 void BioXASSideAppController::initializeBeamline()
 {
 	BioXASSideBeamline::bioXAS();
+	connect( BioXASSideBeamline::bioXAS()->ge32ElementDetector(), SIGNAL(connected(bool)), this, SLOT(onGeDetectorConnectedChanged()) );
+	onGeDetectorConnectedChanged();
+
+	setupScanConfigurations();
```

Comment:
This is probably a mistake on my part in trying to merge in the recent changes from master. I'll remove this call.